Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.1
```

Comment:
I'm on Mojave, so I can't run 5.2. Had to set to 5.1 for development. We can bump again before merging